ui/page/governance/proposal_details_page.go

```diff
@@ -98,6 +100,13 @@ func (pg *ProposalDetails) ID() string {
 func (pg *ProposalDetails) OnNavigatedTo() {
 	pg.ctx, pg.ctxCancel = context.WithCancel(context.TODO())
 	pg.listenForSyncNotifications()
+
+	if pg.ProposalNotificationListener == nil {
+		pg.ProposalNotificationListener = listeners.NewProposalNotificationListener(make(chan wallet.Proposal, 4))
```

Comment:
I wouldn't mind doing this always, regardless of whether `pg.ProposalNotificationListener` is nil or not. It's practically the same thing and saves 3 lines of code.